partials/shop-invoicetotals.htm

```diff
@@ -2,21 +2,23 @@
 {% set itemTotalNoTax = 0 %}
 {% for shipment in invoice.shipments %}
   {% for item in shipment.shipmentItems %}
-    {% set itemTotalNoTax = itemTotalNoTax + item.original_price %}
+    {% set itemTotalNoTax = itemTotalNoTax + item.original_price * item.quantity %}
```

Comment:
`item.original_price` shows the un-discounted price.

`item.price` shows the discounted price.

`item.total` shows the `item.price*quantity`, so you can just use:

`{% set itemTotalNoTax = itemTotalNoTax + item.total %}` here